parsec_web/CHANGELOG.md

```diff
@@ -0,0 +1,200 @@
+# Changelog

```

Comment:
this changelog should be simple, keep the complex details to the MAIN changelog of the parsec_flutter.

https://github.com/oxeanbits/parsec_flutter/blob/main/parsec_linux/CHANGELOG.md

Reply:
Done!